docs/docker-hub/userguide.md

```diff
@@ -33,7 +33,7 @@ you can look at your profile page on [Docker Hub](https://hub.docker.com).
 
 ## Pulling, running and building images
 
-You can find more information on [working with Docker images](../userguide/dockerimages.md).
+You can find more information on [working with Docker images](/userguide/dockerimages/).
```

Comment:
oh, you're using tabs, but the rest of the file uses spaces. we should fix that up to avoid the wrath of Tianon 😈 (I'll create a follow up PR if we don't want that in this PR)